.github/workflows/github-ci.yml

```diff
@@ -466,6 +469,13 @@ jobs:
         path: testpackage-output.tar.gz
       # Note: Testpackage output is further processed in the pr_report.yml workflow
       # (to produce Checks against pull requests)
+    - name: Run fluxfunction test
+      run: |
+        chmod +x $GITHUB_WORKSPACE/fluxfunction
+        srun --job-name fluxtest -M carrington -N 1 -c 1 --mem=1G bash -c "module purge; module load GCC/13.2.0; module load OpenMPI/4.1.6-GCC-13.2.0 ; module load PMIx/4.2.6-GCCcore-13.2.0; module load PAPI/7.1.0-GCCcore-13.2.0; $GITHUB_WORKSPACE/fluxfunction testpackage/run_*/Magnetosphere_small/bulk.0000001.vlsv equatorial.bin"
+        srun --job-name fluxtest -M carrington -N 1 -c 1 --mem=1G bash -c "module purge; module load GCC/13.2.0; module load OpenMPI/4.1.6-GCC-13.2.0 ; module load PMIx/4.2.6-GCCcore-13.2.0; module load PAPI/7.1.0-GCCcore-13.2.0; $GITHUB_WORKSPACE/fluxfunction testpackage/run_*/Magnetosphere_polar_small/bulk.0000001.vlsv polar.bin"
+        diff -q equatorial.bin /proj/group/spacephysics/vlasiator_testpackage/CI_reference/equatorial.bin
```

Comment:
OK so this now reported nothing as expected, but if it were saying there's a diff it wouldn't be very prominent. How do we raise that?

Reply:
pipe the output into a variable, and if the variable is acceptable, touch a file? then can have a yaml action checking for that file and failing on file not found?
@ursg might have a more elegant solution.